generic/tests/netperf.py

```diff
@@ -120,19 +121,23 @@ def mtu_set(mtu):
             ssh_cmd(server_ctl, server_mtu_cmd % (ethname, mtu))
         elif params.get("os_type") == "windows":
             connection_id = utils_net.get_windows_nic_attribute(
-                            server_ctl, "macaddress", mac, "netconnectionid")
+                server_ctl, "macaddress", mac, "netconnectionid")
             ssh_cmd(server_ctl, server_mtu_cmd % (connection_id, mtu))
 
         error_context.context("Changing the MTU of client", logging.info)
-        ssh_cmd(client, client_mtu_cmd % (params.get("client_physical_nic"), mtu))
+        ssh_cmd(client, client_mtu_cmd
+                % (params.get("client_physical_nic"), mtu))
 
         netdst = params.get("netdst", "switch")
         host_bridges = utils_net.Bridge()
         br_in_use = host_bridges.list_br()
         if netdst in br_in_use:
             ifaces_in_use = host_bridges.list_iface()
             target_ifaces = list(ifaces_in_use + br_in_use)
-        error_context.context("Change all Bridge NICs MTU to %s" % mtu, logging.info)
+        if params.get("nettype_nic1") == "macvtap":
```

Comment:
Please do not hardcode the `nic1` here, instead, use vm's method to get the type.

Reply:
I have updated and pass the test.
JOB LOG    : /root/avocado/job-results/job-2018-03-22T03.49-31b7d8a/job.log
 (1/2) smp_8.4096m.repeat1.repeat1.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.5.x86_64.myprovider.netperf.jumbo.host_guest: PASS (239.42 s)
 (1/2) smp_8.4096m.repeat1.repeat1.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.5.x86_64.myprovider.netperf.default.host_guest: PASS (323.33 s)